mewarpx/mewarpx/mwxrun.py

```diff
@@ -298,6 +360,22 @@ def _set_grid_params(self):
         else:
             raise ValueError("Unrecognized type of pywarpx.picmi Grid.")
 
+    def _print_grid_params(self):
+        """Function to print out the grid parameters."""
+        if self.geom_str == 'RZ':
+            logger.info(
+                f"Creating grid with nr={self.nr}, ny={self.ny}, nz={self.nz} "
```

Comment:
Is ny defined?

Reply:
Yeah, similarly as I commented above: in `mwxrun.py` we have `self.nx = self.ny = self.nr = self.grid.nr`